src/pytest_markdown_docs/plugin.py

```diff
@@ -182,6 +182,26 @@ def extract_code_blocks(
             prev = code_block
 
 
+def parse_block_info(block_info: str) -> typing.List[str]:
+    block_info = block_info.strip()
+    # The default `python continuation` format is not compatible with Material for Mkdocs.
+    # But, PyMdown Superfences has a special brace format to add options to code fence blocks: `{.<lang> <option1> <option2>}`.
+    if block_info.startswith("{"):
+        block_info = block_info.strip("{").strip("}")
```

Comment:
You can specify multiple chars within `str.split()` and thus only call it once 😉 
```suggestion
        block_info = block_info.strip("{}")
```